crowdaction/main.go

```diff
@@ -0,0 +1,294 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+const (
+
+	//date format
+	dateFormat = "20060102"
+
+	//for the time being let's consider 30 days how far in future we retrieve crowdactions
+	//could use a parameter store here
+	//example: current date = 2021.12.01 => retrieve up to and including crowdaction end date=2021.12.30
+	dateLimit = 30
```

Comment:
Please create a new task for pagination in Jira and shortly explain what needs to be done.

Reply:
https://collaction.atlassian.net/browse/CAN-99